db/version_set.h

```diff
@@ -802,6 +802,10 @@ class VersionSet {
 
   uint64_t current_next_file_number() const { return next_file_number_.load(); }
 
+  uint64_t latest_min_log_number_to_keep() const {
```

Comment:
What is the reason for not following the naming convention for functions?

Reply:
https://google.github.io/styleguide/cppguide.html#Function_Names